client/components/info-tooltip/style.scss

```diff
@@ -2,3 +2,33 @@
 	cursor: help;
 	color: $gray-dark;
 }
+
+&.tooltip.popover.info-tooltip__container {
+	.popover__inner {
```

Comment:
`info-tooltip__container` was previously called `wc-connect-popover`, I moved the styles from [the main style file](https://github.com/Automattic/woocommerce-services/pull/1051/files#diff-71e5f6f03e3a279beddfc328664288eaL433) to this, component-specific file.